etl/scripts-ccao-data-warehouse-us-east-1/housing/housing-ari_index.py

```diff
@@ -0,0 +1,48 @@
+import os
+import tempfile
+from datetime import datetime
+from io import BytesIO
+
+import boto3
+import pandas as pd
+from dotenv import load_dotenv
+
+# Set up the S3 client
+s3 = boto3.client("s3")
+temp_file = tempfile.NamedTemporaryFile(suffix=".xlsx", delete=False)
+
+load_dotenv("etl/.Renviron")
+AWS_S3_RAW_BUCKET = os.getenv("AWS_S3_RAW_BUCKET")[5:]
+file_key = os.path.join("housing", "ari_index", "2023-ARI.xlsx")
+
+s3.download_file(AWS_S3_RAW_BUCKET, file_key, temp_file.name)
+
+# Use pandas to read the Excel file, skipping the first two rows
+data = pd.read_excel(temp_file.name, skiprows=2, engine="openpyxl")
+current_year = datetime.now().year
+data["year"] = current_year
```

Comment:
We don't know if/how often the file will be updated. The creation of the year was to upload to allow it to forward-fill.